screens/CreateGame.js

```diff
@@ -124,55 +172,27 @@ function CreateGame({ navigation }) {
       setErrorMessage("Please select a location");
       return;
     }
-    //convert chosen date to epoch timestamp in seconds
-    epochDate = Math.floor(chosenDate.getTime() / 1000);
-    //check if specified datetime is in the past
-    epochNow = Math.floor(Date.now() / 1000);
-    if (epochDate <= epochNow) {
-      setShowError(true);
-      setErrorMessage("Please select a valid future date and time");
-      return;
-    }
-
+    
     // START making API calls
     setLoading(true);
-    const location = await getLocation(gameLocation);
-    // if location is null, there was an error getting the location. Error logged in getLocation()
-    if (!location) {
-      setLoading(false);
-      return;
-    }
-    const organizer = await getPlayer();
-    // If there is no organizer, there was a problem and an error was logged
-    if (!organizer) {
-      setLoading(false);
-      return;
-    }
-    //todo: this should eventually be list of players that organizer invites, for now it is just all players in the db
-    // const invitedPlayers = await getAllPlayers();
-    // invitedPlayers.push(organizer); // add organizer to invited players so that they get added in GamePlayer
 
-    const gameInfo = {
-      // if gamename is empty, set it
-      name: gameName ? gameName : "Pickup game at " + location.name,
-      description: gameDescription,
-      date: epochDate,
-      // if the skill level is undefined, set it to ANY
-      skillLevel: gameSkillLevel ? gameSkillLevel : SkillLevel.ANY,
-      organizer: organizer,
-      location: location,
-    };
-    let newGame = await storeGame(gameInfo);
-    if (newGame == null) {
+    let newGame = await storeGame();
+    if (newGame == null || newGame == undefined) {
       setLoading(false);
+      setErrorMessage("Error creating game.")
```

Comment:
Dont set an error message here, we already set one inside storeGame() if something goes wrong.